model_compression_toolkit/ptq/pytorch/quantization_facade.py

```diff
@@ -64,6 +65,8 @@ def pytorch_post_training_quantization_experimental(in_module: Module,
             core_config (CoreConfig): Configuration object containing parameters of how the model should be quantized, including mixed precision parameters.
             target_platform_capabilities (TargetPlatformCapabilities): TargetPlatformCapabilities to optimize the PyTorch model according to. `Default PyTorch TPC <https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/core/tpc_models/pytorch_tp_models/pytorch_default.py>`_
             new_experimental_exporter (bool): Whether exporting the quantized model using new exporter or not (in progress. Avoiding it for now is recommended).
+            model_leaf_layers (list): List of the module's custom layers, these layers shouldn't be divided into their
```

Comment:
List of the module's custom layers. These layers shouldn't be divided into their submodules. Please note that their quantization will not be optimized.